api/src/Middleware/CiabMiddleware.php

```diff
@@ -14,6 +14,19 @@ class CiabMiddleware
 {
 
 
+    protected function errorResponse(Request $request, Response $response, string $message, $status, int $code):  Response
+    {
+        $result = [
+        'type' => 'error',
+        'code' => $code,
+        'status' => $status,
+        'message' => $message,
```

Comment:
We probably don't need to do this, actually. Including the `message` would be good, but I think it's already implied that we're dealing with an error, and the HTTP Status code/header also contains this information without needing to include it in the response body.